app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -524,7 +527,7 @@ def find_organization_from_product
     end
 
     def find_content
-      @content = @repository.units_for_removal(params[:ids])
+      @content = @repository.units_for_removal(params[:ids], params[:content_type])
```

Comment:
This needs to fall back to the default content type if content_type isn't passed in.  When trying to remove an rpm from a yum repo i get an ISE due to this.

Reply:
actually the error is a bit different, maybe related?
```
2019-07-30T15:47:08 [E|app|53059a12] NoMethodError: undefined method `table_name' for #<Katello::RepositoryType::ContentType:0x00007f06ccd35c00>
 | /home/vagrant/git/katello/app/models/katello/repository.rb:541:in `units_for_removal'
 | /home/vagrant/git/katello/app/controllers/katello/api/v2/repositories_controller.rb:530:in `find_content'
 | /home/vagrant/.gem/ruby/gems/activesupport-5.2.1/lib/active_support/callbacks.rb:426:in `block in make_lambda'
 | /home/vagrant/.gem/ruby/gems/activesupport-5.2.1/lib/active_support/callbacks.rb:179:in `block (2 levels) in halting_and_conditional'
```

Reply:
actually this matches your test errors! so you may already be working on fixing it